doc/api/http.markdown

```diff
@@ -103,7 +103,7 @@ of these values set to their respective defaults.
 To configure any of them, you must create your own [`http.Agent`][] object.
 
 ```javascript
-var http = require('http');
+const http = require('http');
```

Comment:
I'm not sure we really want to change `var` to `const` in the examples in general. `const` requires strict mode, so you need to know to include `'use strict'`. Using `var` is more widely understood and doesn't require the extra knowledge about `'use strict'`. It contradicts our own internal informal style (where we seem to be strongly favoring `const` in these situations) but I actually don't think that's relevant for the docs.